ports/libfv/vcpkg.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "libfv",
```

Unfortunately, `libfv` has a high chance of collision and confusion in the future (too short). The GitHub repo doesn't need to be renamed, but the package in the vcpkg port namespace needs to be prefixed with your GitHub organization (it will still show up in `vcpkg search libfv`).

```suggestion
  "name": "fawdlstty-libfv",
```